src/components/DataTable/DataTable.stories.tsx

```diff
@@ -570,10 +574,9 @@ export const InteractiveTable = () => {
 
 	return <Component />;
 };
-InteractiveTable.storyName = 'InteractiveTable';
```

Comment:
Storybook automatically adds titles, including spacing, and declaring the storyName like this causes the story title to appear exactly like this without spacing.